connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinator.java

```diff
@@ -50,7 +50,6 @@
  * to workers.
  */
 public class WorkerCoordinator extends AbstractCoordinator implements Closeable {
-    // Currently doesn't support multiple task assignment strategies, so we just fill in a default value
     public static final String DEFAULT_SUBPROTOCOL = "default";
```

Comment:
It seems ```DEFAULT_SUBPROTOCOL``` is used by testing only. Should we move it to test code?

Reply:
The definition has moved into: 
https://github.com/apache/kafka/blob/trunk/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/ConnectProtocolCompatibility.java#L45

We should use `EAGER.protocol()` to replace the usage of this member field variable and then remove it.

Reply:
Sounds good, done.